### PR TITLE
Tenant management UI

### PR DESCRIPTION
**Pull request by gaarf**
This PR is an initial implementation of #450 and includes:
- new "tenants" tab in nav for admins, re-using the services icon for now
- tenant list with workers/queues display (does not live-update yet)
- tenant delete button
- tenant create/edit form

and as a bonus:
- CSS tweak for the navbar on small screens

### CODE REVIEW

File: ui/client/js/tenants.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
+(function() {
+
+var app = angular.module('loom', [], ['$interpolateProvider', function ($interpolateProvider) {
+  $interpolateProvider.startSymbol('[[');
+  $interpolateProvider.endSymbol(']]');
+}]);
+
+
+
```
Comment (wolf31o2) [inline ui/client/js/tenants.js:9]: Is this much whitespace normal?

File: ui/client/templates/partials/left-menu.html [hunk 2]
```diff
@@ -13,6 +13,10 @@
  See the License for the specific language governing permissions and
  limitations under the License.
 -->
+<li class="overview {% if activeTab == 'tenants' %}active{% endif %} nav-item" id="nav-tenants-container">
+  <a class="nav-icon nav-tenants" href="/tenants"></a>
+  <a class="nav-link" id="nav-tenants" href="/tenants">Tenants</a>
+</li>
```
Comment (wolf31o2) [inline ui/client/templates/partials/left-menu.html:21]: This is fine with me, but @nitinmotgi may have something to say about it being first.

File: ui/client/js/tenants.js (hunk 1 above)
Comment (gaarf) [inline ui/client/js/tenants.js:9]: do you mean before/after statement or indentation?

**Comment by albertshau** [comment]
The tenant tab should only be visible to somebody with superadmin privileges (currently determined by tenant=superadmin, user=admin). Otherwise lgtm

**Comment by albertshau** [comment]
:+1:

**Pull request merged**